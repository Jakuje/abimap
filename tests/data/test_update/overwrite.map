# This map file was created with symbol_version.py

LIBTC4_1_0_0
{
    global:
        another_symbol;
        symbol;
    local:
        *;
} ;

