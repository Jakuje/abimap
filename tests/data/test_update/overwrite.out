# This map file was automatically updated

LIBTC4_2_0_0
{
    global:
        new_symbol;
        symbol;
    local:
        *;
} ;

